### PR TITLE
Adds an alt-click prompt to the descriptions of various objects

### PR DESCRIPTION
**Pull request by PKPenguin321**
Includes:
1. The detective's revolver (and any gun that can be reskinned)
2. Washing machines
Feel free to suggest more

A lot of people have no idea how to start the washing machine, and many people thought that the detective revolver reskinning got removed. Now there will be actual feedback on how to use them.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Also I feel that it would look nicer if it had a line break and was blue text but was too lazy to figure out how to do that, it anybody knows hit me up

**Comment by Supermichael777** [comment]
-tg-station/interface/stylesheet.dm
 use span class and then the name of the style you want to use

like "<span class='userdanger'>[M] has slashed at [src]!</span>" produces some bold red text at 3 font

<span class='info'>your text</span> should give you a nice dark blue

**Comment by KazeEspada** [comment]
You can add line breaks to outputs because it accepts html, including linebreaks. But supermichaels way works best.

**Comment by optimumtact** [comment]
You know you can left click on the washing machine to start it right?

**Comment by PKPenguin321** [comment]
that opens the door and spills out all the clothing iirc

**Comment by CosmicScientist** [comment]
Could the revolver not have the message to reskin it after it's reskinned or would I be asking too much?

What about lockers so I can be annoying and think I have an excuse to guilt people into locking them after use?

**Comment by PKPenguin321** [comment]
Now that I know how to use the examine proc, removing the message after the skin is applied would be easy

I'll add lockers next commit

**Comment by CosmicScientist** [comment]
Yay I hope it'll work eventually!

File: code/game/machinery/washing_machine.dm [hunk 1]
```diff
@@ -17,6 +17,10 @@
 	var/gibs_ready = 0
 	var/obj/crayon
 
+/obj/machinery/washing_machine/examine(mob/user)
+	..()
+		user << "<span class='italicnotice'>Alt-click it to start a wash cycle.</span>"
```
Comment (MrPerson) [inline code/game/machinery/washing_machine.dm]: This line is indented too far

File: code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm [hunk 2]
```diff
@@ -5,3 +5,7 @@
 	icon_state = "secure"
 	health = 200
 	secure = 1
+
+/obj/structure/closet/secure_closet/examine(mob/user)
+	..()
+		user << "<span class='italicnotice'>Alt-click it to toggle the lock.</span>"
```
Comment (MrPerson) [inline code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm]: same here

Comment (PKPenguin321) [inline code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm]: Ah thanks fam, I'll fix this next time I can get at a computer

**Comment by PKPenguin321** [comment]
@Cheridan @KorPhaeron @duncathan @RemieRichards @ChangelingRain @Jordie0608 this is pretty much good to go

File: code/modules/projectiles/guns/projectile/revolver.dm [hunk 3]
```diff
@@ -87,6 +87,10 @@
 /obj/item/weapon/gun/projectile/revolver/examine(mob/user)
 	..()
 	user << "[get_ammo(0,0)] of those are live rounds."
+	if(unique_reskin && !reskinned)
+		user << "<span class='italicnotice'>Alt-click it to reskin it.</span>"
+	if(unique_rename)
```
Comment (KorPhaeron) [inline code/modules/projectiles/guns/projectile/revolver.dm]: Shouldn't this bit be on all guns? All guns have a unique rename bit.

Comment (PKPenguin321) [inline code/modules/projectiles/guns/projectile/revolver.dm]: you're right

**Comment by PKPenguin321** [comment]
okay NOW it should be really really good to go

**Comment by PKPenguin321** [comment]
okay so i actually tested and noticed a couple of things
1. the font style i added isn't actually blue or italic like i intended i need to fix that but i dont even know what im doing with the font style part help
2. lockers already have a message, probably defined somewhere down the line
3. washing machine works fine (aside from the font not being the intended style like i already mentioned)

**Comment by Razharas** [comment]
DO something with blue and italic andf ill merge @PKPenguin321

**Comment by PKPenguin321** [comment]
I tried to but it didn't work and I don't know how to fix it send help

**Comment by KorPhaeron** [comment]
@PKPenguin321 

Just remove the italic thing

**Comment by PKPenguin321** [comment]
Will do later today I guess

**Comment by KorPhaeron** [comment]
@PKPenguin321 

reeee

**Comment by PKPenguin321** [comment]
Haha I forgot
Tomorrow most likely

**Comment by PKPenguin321** [comment]
so assuming this passes travis it should be good to go, but i warn that i haven't had time to actually test this in game so it might not look the way i want it? not sure

**Pull request merged**